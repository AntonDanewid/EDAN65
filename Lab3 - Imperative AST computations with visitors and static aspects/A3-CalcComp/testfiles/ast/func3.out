Program
  List
    FuncDecl
      IntType
      IdDecl
      List
        Param
          IntType
          IdDecl
        Param
          IntType
          IdDecl
      Block
        List
          ReturnStmt
            IdUseExpr
    FuncDecl
      IntType
      IdDecl
      List
      Block
        List
          DeclStmt
            IntType
            IdDecl
          DeclStmt
            IntType
            IdDecl
          IfStmt
            GT
              IdUseExpr
              IdUseExpr
            Block
              List
                ReturnStmt
                  IdUseExpr
            Opt
