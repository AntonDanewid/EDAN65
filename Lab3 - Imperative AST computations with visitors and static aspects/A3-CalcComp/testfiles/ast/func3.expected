Program
  List
    FuncDecl
      IdDecl
      List
        IdDecl
        IdDecl
      Block
        List
          ReturnStmt
            IdUseExpr
    FuncDecl
      IdDecl
      List
      Block
        List
          DeclStmt
            IdDecl
          DeclStmt
            IdDecl
          IfStmt
            GT
              IdUseExpr
              IdUseExpr
            Block
              List
                ReturnStmt
                  IdUseExpr
            Opt
